.github/workflows/release.yml

```
@@ -184,7 +184,7 @@ jobs:
 
       # Create the GitHub release notes, upload artifact backups to S3, publish homebrew recipe
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@7ec5c2b0c6cdda6e8bbb49444bc797dd33d74dd8 # v5.0.0
+        uses: goreleaser/goreleaser-action@286f3b13b1b49da4ac219696163fb8c1c93e1200 # v6.0.0
         with:
           distribution: goreleaser
           version: latest
```

version needs to be updated
```suggestion
          version: "~> v2"
```
https://goreleaser.com/blog/goreleaser-v2/#github-action

---

> If you use our [GitHub Action](https://github.com/goreleaser/goreleaser-action), the latest version (v6.0.0) should use `~> v2` by default if your version option is either empty or latest.

---

> I recommend you update it:

```yaml
# .github/workflows/release.yml

# ...
jobs:
  goreleaser:
    # ...
    - uses: goreleaser/goreleaser-action@v6
      with:
        distribution: goreleaser # or 'goreleaser-pro'
        version: "~> v2" # or 'latest', 'nightly', semver
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        # GORELEASER_KEY: ${{ secrets.GORELEASER_KEY }} # if using goreleaser-pro
```